mattermost/mattermost-mobile
# MM-15224 Initial config for RN UI Automation - Appium for Android app

#### Summary
This is the initial config for RN UI Automation - Appium for Android app.

How to run E2E test:
1. `make build-android` - build Android app and verify that the `Mattermost.apk` is available at root directory on successful build.
2. `npm run e2e:android` - npm script to run E2E for Android

Packages used:
- `appium@1.12.1`
- `wd@1.11.2`
- `tape@4.10.1`
- `tap-spec@5.0.0`

#### Ticket Link
Jira ticket: [MM-15224](https://mattermost.atlassian.net/browse/MM-15224)

#### Checklist

#### Device Information
This PR was tested on: [Android]
